tests/testapp/ui.json

```diff
@@ -1988,7 +1988,7 @@
                 "eventType": "ss-change",
                 "stateRef": "name"
             },
-            "visible": ""
+            "visible": false
```

Comment:
This seems to be a typo: I've verified that actual UI update resolves to "false" for components with disabled visibility. And, this failed Pydantic validation, so I had to change it.